staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -564,3 +565,70 @@ type MemoryPreferenceRequirement struct {
 	// Minimal amount of memory required by the preference.
 	Guest resource.Quantity `json:"guest"`
 }
+
+type ControllerRevisionUpgradePhase string
+
+const (
+	UpgradeRunning   ControllerRevisionUpgradePhase = "Running"
+	UpgradeSucceeded ControllerRevisionUpgradePhase = "Succeeded"
+	UpgradeFailed    ControllerRevisionUpgradePhase = "Failed"
+)
+
+// ControllerRevisionUpgrade encapsulates a specific upgrade of a stashed ControllerRevision instance type object to the latest available version
+//
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
+type ControllerRevisionUpgrade struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec *ControllerRevisionUpgradeSpec `json:"spec,omitempty"`
+	// +nullable
+	Status *ControllerRevisionUpgradeStatus `json:"status,omitempty"`
```

Comment:
I forgot to comment on this in my last review.
Usually, in other objects, `Spec` and `Status` fields are not pointers. Can you remove the pointers?